{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py

```diff
@@ -1,4 +1,7 @@
-"""{{cookiecutter.description}}"""
+"""
+{{cookiecutter.project_name}}
+{{cookiecutter.description}}
+"""
 
 # Add imports here
 from .{{cookiecutter.first_module_name}} import *
```

Comment:
Do we really want to encourage `*` imports?

Reply:
No, I changed it to `canvas`.